docs/diracdoctools/scripts/dirac-docs-concatenate-diraccfg.py

```diff
@@ -3,9 +3,7 @@
 from __future__ import absolute_import
 from __future__ import division
 from __future__ import print_function
-import sys
 
 from diracdoctools.cmd.concatcfg import run
-from diracdoctools.Config import CLParser
 
-sys.exit(run(**(CLParser().optionDict())))
+run()
```

Comment:
So now there is no way to configure this scrip any more? And it always exits with status code 0?